cumulus/node/README.md

```diff
@@ -7,103 +7,66 @@
 
 ```bash
-$ cargo build --release
+$ cargo build --release --bin subspace-farmer --bin subspace-node
 ```
 
 ### Spin up a local testnet
 
-1. Run a primary node.
+1. Run a primary node with an executor Alice running in authority mode.
 
 ```bash
-$ ./target/release/subspace-node --dev -d tmp --log=txpool=trace,gossip::executor=trace
-2022-04-16 11:40:57.269  INFO main sc_cli::runner: Subspace
-2022-04-16 11:40:57.269  INFO main sc_cli::runner: ✌️  version 0.1.0-ab4a59751
-2022-04-16 11:40:57.269  INFO main sc_cli::runner: ❤️  by Subspace Labs <https://subspace.network>, 2021-2022
-2022-04-16 11:40:57.269  INFO main sc_cli::runner: 📋 Chain specification: Development
-2022-04-16 11:40:57.269  INFO main sc_cli::runner: 🏷  Node name: harmonious-feast-4450
-2022-04-16 11:40:57.269  INFO main sc_cli::runner: 👤 Role: AUTHORITY
-2022-04-16 11:40:57.269  INFO main sc_cli::runner: 💾 Database: RocksDb at tmp/chains/dev/db/full
-2022-04-16 11:40:57.269  INFO main sc_cli::runner: ⛓  Native runtime: subspace-100 (subspace-1.tx1.au1)
-2022-04-16 11:40:57.484  INFO main sc_service::client::client: 🔨 Initializing Genesis block/state (state: 0x12cf…0724, header-hash: 0x42de…9335)
-2022-04-16 11:40:57.635  INFO main subspace: Starting archiving from genesis
-2022-04-16 11:40:57.667  INFO main subspace: Archiving already produced blocks 0..=0
-2022-04-16 11:40:57.756  WARN main sc_service::config: Using default protocol ID "sup" because none is configured in the chain specs
-2022-04-16 11:40:57.757  INFO main sub-libp2p: 🏷  Local node identity is: 12D3KooWMQYGEy2eEcci2RoLZSiBdMj34UjipYR4GryjHkNsDXYW
-2022-04-16 11:40:57.759  INFO main subspace: 🧑‍🌾 Starting Subspace Authorship worker
-2022-04-16 11:40:57.768  INFO main sc_sysinfo: 💻 Operating system: linux
-2022-04-16 11:40:57.768  INFO main sc_sysinfo: 💻 CPU architecture: x86_64
-2022-04-16 11:40:57.768  INFO main sc_sysinfo: 💻 Target environment: gnu
-2022-04-16 11:40:57.768  INFO main sc_sysinfo: 💻 CPU: AMD Ryzen 9 5900X 12-Core Processor
-2022-04-16 11:40:57.768  INFO main sc_sysinfo: 💻 CPU cores: 12
-2022-04-16 11:40:57.768  INFO main sc_sysinfo: 💻 Memory: 64252MB
-2022-04-16 11:40:57.768  INFO main sc_sysinfo: 💻 Kernel: 5.13.0-35-generic
-2022-04-16 11:40:57.768  INFO main sc_sysinfo: 💻 Linux distribution: Ubuntu 20.04.4 LTS
-2022-04-16 11:40:57.768  INFO main sc_sysinfo: 💻 Virtual machine: no
-2022-04-16 11:40:57.768  INFO main sc_service::builder: 📦 Highest known block at #0
-2022-04-16 11:40:57.768  INFO main parity_ws: Listening for new connections on 127.0.0.1:9944.
-2022-04-16 11:40:57.811  INFO ThreadId(84) parity_ws::io: Accepted a new tcp connection from 127.0.0.1:58000.
-2022-04-16 11:41:02.768  INFO tokio-runtime-worker substrate: 💤 Idle (0 peers), best: #0 (0x42de…9335), finalized #0 (0x42de…9335), ⬇ 0 ⬆ 0
-
+$ ./target/release/subspace-node --dev -d tmp --log=runtime=debug -- --alice --dev --port 40333 --log=cirrus::executor=trace,txpool=trace,gossip=trace --rpc-port 8845 --ws-port 8846
```

Comment:
Should we put tracing level logs into examples though?

Reply:
I don't mind as it's a local network anyway. I can just remove the log if you prefer it to be simpler.

Reply:
These instructions are for anyone to run and I think they'll be surprise by the number of things printed. I get that you need it for your work of course.

Reply:
Updated.